.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Feature request
+about: Suggest a new feature in rust-libp2p.
+---
+
+<!--
+Note: If you'd like to suggest an feature related to libp2p but not specifically related to the rust implementation, please file an issue at https://github.com/libp2p/libp2p instead. Even better, create a new topic on the forums (https://discuss.libp2p.io).
+-->
+
+## Description
+
+<!-- Briefly describe the feature that you are requesting.-->
+
+## Motivation
+
+<!-- Explain why this feature is needed.-->
+
+## Requirements
+
+<!-- Write a list of what you want this feature to do.-->
+
+1.
+2.
+3.
+
+## Open questions
+
+<!-- Optional: use this section to ask any questions that are related to the feature.-->
+
+## Are you planning to do it yourself in a pull request?
+
+<!--Any contribution is greatly appreciated. We are more than happy to provide help on the process.-->
```

Comment:
This is cool! I like emphasizing the fact that this project is open to external contributions.